### DISCUSSION TITLE
Run failed: Node 图文回复 run failed: 'FileVar' object has no attribute 'markdown'

### DISCUSSION THREAD

**Discussion by oocsoo** [langgenius/dify#7381] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

Run failed: Node 图文回复 run failed: 'FileVar' object has no attribute 'markdown'

 
![微信图片_20240819000212](https://github.com/user-attachments/assets/3f7901f5-5fe6-407c-b7d4-039eb5d270c5)

**Comment by crazywoola** [comment]
Fixed in v0.7.1
